examples/autoquality/autoquality_usage.ipynb

```diff
@@ -0,0 +1,709 @@
+{
```

Comment:
<p>Add title for this part, like</p><p> </p><pre class="ql-syntax" spellcheck="false"># Training and base pool setup
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Toloka/toloka-kit/pull/75/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>